data-collectors/ciphertrace/src/main/java/com/unboundtech/casp/datacollector/cyphertrace/sample/Main.java

```diff
@@ -154,7 +155,12 @@ public static void calcRiskForSignRequests(){
 
             TransactionHandler handler = getTransactionHandler(signRequest);
 
-            List<DetailedTransaction> detailedTransactions = handler.decode(signRequest.dataToSign, signRequest.rawTransactions, signRequest. publicKeys, Collections.emptySet());
+            List<DetailedTransaction> detailedTransactions = null;
+            try {
+                detailedTransactions = handler.decode(signRequest.dataToSign, signRequest.rawTransactions, signRequest. publicKeys, Collections.emptySet());
+            } catch (BadTransactionException e) {
+                e.printStackTrace();
```

Comment:
the exception  needs to written to log, not to console.
also on error, the code needs to return and wait for the next interation. It cannot continue parsing the TX.

Reply:
i think your code is not updated...can u try to pull it again

Reply:
done